src/Chapter3.hs

```diff
@@ -374,6 +379,32 @@ after the fight. The battle has the following possible outcomes:
 
 -}
 
+data Knight = Knight
+  { knightName      :: String
+  , knightHealth :: Int
+  , knightAttack :: Int
+  , knightGold :: Int
+  } deriving (Show)
+
+data Monster = Monster
+  { monsterName      :: String
```

Comment:
Any reasons to add a certain amount of spaces just to the first field of the record?
I am just curious 🙂

Reply:
Just a bad copypasta 😅